tools/PI/DevHome.PI/BarWindow.cs

```diff
@@ -76,6 +76,8 @@ internal void SetRequestedTheme(ElementTheme theme)
 
     public void NavigateTo(Type viewModelType) => _horizontalWindow.NavigateTo(viewModelType);
 
+    public void NavigateToSettings(string settingsPage) => _horizontalWindow.NavigateToSettings(settingsPage);
```

Comment:
Reading this I assumed it meant Dev Home settings, but on closer inspection it seemed to be PI-specific settings. Consider making that more clear in the method and/or parameter names.

Reply:
Changed to NavigateToPiSettings